packages/types/src/clerk.ts

```diff
@@ -645,6 +645,16 @@ export type ClerkOptions = ClerkOptionsNavigation &
         };
 
     sdkMetadata?: SDKMetadata;
+
+    /**
+     * Toggle experimental behaviour
```

Comment:
```suggestion
     * Enable experimental flags to gain access to new features. These flags are not guaranteed to be stable and may change drastically in between patch or minor versions.
```

If/When we have docs on this prop we should link to it here